src/OpenTelemetry/Metrics/MetricReaderExt.cs

```diff
@@ -42,7 +42,7 @@ internal Metric AddMetricWithNoViews(Instrument instrument)
             var meterVersion = instrument.Meter.Version;
             var metricName = instrument.Name;
             var metricStreamName = $"{meterName}.{meterVersion}.{metricName}";
-            var instrumentIdentity = new InstrumentIdentity(instrument.Meter, metricName, instrument.Unit, instrument.Description, instrument.GetType());
+            var instrumentIdentity = new InstrumentIdentity(instrument.Meter, metricName, instrument.Unit, instrument.Description, instrument.GetType(), false);
```

Comment:
nit: I personally would pass like `derivedFromView: false`. Just helps with readability to throw the parameter name on there when the thing you are passing doesn't have name (typically `true`, `false`, & `null`).

Reply:
Yep, agree, thanks